package.json

```diff
@@ -35,7 +35,6 @@
     "@angular/http": "^4.1.0"
   },
   "dependencies": {
-    "moment": "^2.9.0"
```

Comment:
So... that's a point I wanted to talk about, thank you for reminding it to me :)

First answer is: yes, moment is still used.

But, I've tried hundreds of combinations (not so much exaggerating) to figure out how I could include dependencies here, make them both ok for the typescript transpiler and the runtime (I can get one or the other working, but not both) and also make them correctly rolled up so that they're included in the bundle files and used by downstream app. I've spent hours stack-overflowing (see for instance: https://stackoverflow.com/questions/39519823/using-rollup-for-angular-2s-aot-compiler-and-importing-moment-js ). I don't know why I can't get it work, but if you could have a look, it would be great!

So, for the time being, it's up to the downstream app to import d3, d3-tip and moment.